apps/publisher/modules/page-decorators.js

```diff
@@ -205,13 +205,13 @@ var pageDecorators = {};
             sortObj.active = false;
             sortObj.sortNext = "+";
             sortObj.sortIcon = "sorting_asc";
-            if(sortable[i].field == sortBy){
-                if(sort == "+"){
-                    sortingListSelected.helpIcon = "sorting_asc";
+            if(sortable[i].field == sortBy) {
+                if(sort == "+") {
```

Comment:
Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.